llvm/lib/Target/Mips/MipsAsmPrinter.cpp

```diff
@@ -72,6 +72,11 @@ using namespace llvm;
 
 extern cl::opt<bool> EmitJalrReloc;
 
+static cl::opt<bool> EmitNanoMipsBalcStubsReloc(
```

Comment:
Option name should reflect the semantics. Is it to enable the optimization or disable it?